input/fsh/DkCoreObservation.fsh

```diff
@@ -1,5 +1,14 @@
+Profile: IPADkCoreObservation
+Parent: DkCoreObservation
+Title: "Danish Core IPA Observation Profiles"
+Description: "HL7 Denmark core profile for IPA compliant observations"
+* subject only Reference(DkCorePatient)
+* extension
+  * url = "http://hl7.org/fhir/StructureDefinition/structuredefinition-compliesWithProfile"
```

Comment:
Shouldn't this be `structuredefinition-imposeProfile` instead? As I understand it, the difference between the two is that the `imposeProfile` extension tells the validator to also validate against the mentioned SD, whereas the `compliesWithProfile`extention declares that all requirements from the mentioned SD is already included in `this` SD - so no extra check is necessary. But in this case we do not repeat all the requirements from IPA.

Another note: These extensions were introduced in R5, so I wouldn't count on a check in an R4 validator - but perhaps they will backport it...

Reply:
(or you can add the extra requirement from the IPA - AFAIR it was restricting subject to DKCorePatient. Perhaps this solution would be more "reader friendly"

Reply:
Its defined in the extension pack - https://hl7.org/fhir/extensions/StructureDefinition-structuredefinition-imposeProfile.html which is already automatically imported into the IG - ... so @jacand if you're good with (after I've changed it to `imposeProfile`) I'll merge if you're good with it?

Reply:
https://github.com/hl7dk/dk-core/pull/132/commits/3c0970599d485cc29d2b5c0aaf73dc7045d95e87